examples/with-react-ga/pages/_app.js

```diff
@@ -1,16 +1,27 @@
-import App from 'next/app'
-import Router from 'next/router'
+import { useEffect } from 'react'
+import { useRouter } from 'next/router'
 import { initGA, logPageView } from '../utils/analytics'
 
-export default class MyApp extends App {
-  componentDidMount() {
+const MyApp = ({ Component, pageProps }) => {
+  const router = useRouter()
+
+  useEffect(() => {
     initGA()
-    logPageView()
-    Router.events.on('routeChangeComplete', logPageView)
-  }
+    // workaround for the issue #11639
+    if (!router.asPath.includes('?')) {
```

Comment:
I don't think the example should be including this 🤔

Reply:
Yes but if someone take this example for production porpuse then it doesn't work correctly in case of query in the url... May I remove the comment or tell me if you'd like to remove completely 👍🏻

Reply:
@valse Add a comment mentioning what you just said, without linking to an issue 👍